README.md
![alt XOOPS CMS](http://xoops.org/images/logoXoops4GithubRepository.png)
##Adminer module for  [XOOPS CMS 2.5.8+](https://xoops.org)
[![Software License](https://img.shields.io/badge/license-GPL-brightgreen.svg?style=flat)](LICENSE)
[![Scrutinizer Code Quality](https://img.shields.io/scrutinizer/g/XoopsModules25x/adminer.svg?style=flat)](https://scrutinizer-ci.com/g/XoopsModules25x/adminer/?branch=master)
[![Codacy Badge](https://api.codacy.com/project/badge/Grade/210665112c1e43e78add5a669f7f6674)](https://www.codacy.com/app/XoopsModules25x/adminer_2)
[![Code Climate](https://img.shields.io/codeclimate/github/XoopsModules25x/adminer.svg?style=flat)](https://codeclimate.com/github/XoopsModules25x/adminer)
[![SensioLabsInsight](https://insight.sensiolabs.com/projects/ddcde09d-0c45-47ec-94de-55f7e5e9d46b/mini.png)](https://insight.sensiolabs.com/projects/ddcde09d-0c45-47ec-94de-55f7e5e9d46b)
[![Latest Pre-Release](https://img.shields.io/github/tag/XoopsModules25x/adminer.svg?style=flat)](https://github.com/XoopsModules25x/adminer/tags/)
[![Latest Version](https://img.shields.io/github/release/XoopsModules25x/adminer.svg?style=flat)](https://github.com/XoopsModules25x/adminer/releases/)

Adminer module for [XOOPS CMS](http://xoops.org) provides SQL Management for XOOPS from inside the Admin Panel

[![Tutorial Available](http://xoops.org/images/tutorial-available-blue.svg)](https://www.gitbook.com/book/xoops/xoops-adminer/) Tutorial: see [GitBook](https://www.gitbook.com/book/xoops/xoops-adminer/).
To contribute to the Tutorial, [fork it on GitHub](https://github.com/XoopsDocs/adminer-tutorial)

[![Translations on Transifex](http://xoops.org/images/translations-transifex-blue.svg)](https://www.transifex.com/xoops)

Please visit us on http://xoops.org

The upcoming "next generation" version of XOOPS CMS is being crafted on GitHub at: https://github.com/XOOPS


changelog.txt
Changelog

version 3.0
	- mise à jour Adminer 3.7.1
	- mise à jour Adminer editor 3.7.1

version 2.9
	- mise à jour Adminer 3.7.0
	- mise à jour Adminer editor 3.7.0

version 2.8
	- mise à jour Adminer 3.6.4
	- mise à jour Adminer editor 3.6.4
	- mise à jour BigDump 0.34b

version 2.7
	- mise à jour Adminer 3.6.3
	- mise à jour Adminer editor 3.6.3

version 2.6
	- mise à jour Adminer 3.6.2
	- mise à jour Adminer editor 3.6.2

version 2.5
	- mise à jour Adminer 3.6.1
	- mise à jour Adminer editor 3.6.1

version 2.4
	- mise à jour Adminer 3.6.0
	- mise à jour Adminer editor 3.6.0

version 2.3
	- mise à jour Adminer 3.5.1
	- mise à jour Adminer editor 3.5.1

version 2.2
	- mise à jour Adminer 3.5.0
	- mise à jour Adminer editor 3.5.0

version 2.1
	- mise à jour Adminer 3.4.0
	- mise à jour Adminer editor 3.4.0

version 2.0
	- mise à jour Adminer 3.3.4
	- mise à jour Adminer editor 3.3.4
	- mise à jour BigDump 0.34b

version 1.9
	- mise à jour Adminer 3.3.3
	- mise à jour Adminer editor 3.3.3

version 1.8
	- mise à jour Adminer 3.3.2
	- mise à jour Adminer editor 3.3.2

version 1.7
	- mise à jour Adminer 3.3.1
	- mise à jour Adminer editor 3.3.1

version 1.6
	- mise à jour Adminer 3.3.0
	- mise à jour Adminer editor 3.3.0

version 1.5
	- mise à jour Adminer 3.2.2
	- mise à jour Adminer editor 3.2.2

version 1.4
	- mise à jour Adminer 3.2.1
	- mise à jour Adminer editor 3.2.1

version 1.3
	- mise à jour Adminer 3.1
	- mise à jour Adminer editor 3.1

version 1.2
	- mise à jour Adminer 3.0.1
	- mise à jour Adminer editor 3.0.1

version 1.1
	- ajout de defines de langue
	- réorganisation de l'architecture et code du module
	
version 1.0
	- initial work (kris_fr)

xoops_version.php
<?php
/**
 * Adminer Module based on Ghost Module
 *
 * You may not change or alter any portion of this comment or credits
 * of supporting developers from this source code or any supporting source code
 * which is considered copyrighted (c) material of the original comment or credit authors.
 * This program is distributed in the hope that it will be useful,
 * but WITHOUT ANY WARRANTY; without even the implied warranty of
 * MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.
 *
 * @copyright           The XOOPS Project http://sourceforge.net/projects/xoops/
 * @license             http://www.fsf.org/copyleft/gpl.html GNU public license
 * @package             Adminer Module
 * @since               2.3.0
 * @author              Kris <http://www.xoofoo.org>
 * @version             $Id $
 **/
$moduleDirName               = basename(__DIR__);
$modversion['name']           = _MI_ADMINER_NAME;
$modversion['version']        = 3.1;
$modversion['description']    = _MI_ADMINER_DESC;
$modversion['credits']        = 'Adminer.org and Alexey Ozerov ';
$modversion['author']         = 'Kris - http://www.xoofoo.org';
$modversion['help']           = 'page=help';
$modversion['license']        = 'GPL see LICENSE';
$modversion['official']       = 0;
$modversion['image']          = 'assets/images/logoModule.png';
$modversion['dirname']        = $moduleDirName;
$modversion['dirmoduleadmin'] = 'Frameworks/moduleclasses/moduleadmin';
$modversion['sysicons16']     = 'Frameworks/moduleclasses/icons/16';
$modversion['sysicons32']     = 'Frameworks/moduleclasses/icons/32';
$modversion['modicons16']     = 'assets/images/icons/16';
$modversion['modicons32']     = 'assets/images/icons/32';
//about
$modversion['module_status']       = 'Beta 1';
$modversion['release_date']        = '2016/07/18';
$modversion['module_website_url']  = 'www.xoops.org';
$modversion['module_website_name'] = 'XOOPS';
$modversion['min_php']             = '5.5';
$modversion['min_xoops']           = '2.5.8';
$modversion['min_admin']           = '1.2';
$modversion['min_db']              = array(
    'mysql'  => '5.0.7',
    'mysqli' => '5.0.7'
);

// Menu
$modversion['hasAdmin']    = 1;
$modversion['system_menu'] = 1;
$modversion['adminindex']  = 'admin/index.php';
$modversion['adminmenu']   = 'admin/menu.php';

// Templates
$modversion['templates'][1]['file']        = $moduleDirName . '_index.tpl';
$modversion['templates'][1]['description'] = '';
